### DISCUSSION TITLE
Replace Strings

### DISCUSSION THREAD

**Discussion by imjustprism** [WerWolv/ImHex#1575] [Get Help]
Is there any method by which I can replace a specific string? And, for instance, all occurrences, specifying the string I want to search for and what I want to replace it with?

**Comment by WerWolv** [comment]
Hey! Yes you can use the Find View. Click on the elements you want to replace to select them (or use CTRL + A for all), then right click one and choose replace there.

**Reply by imjustprism** [reply]
Oh yes, I tried this, but it replaces the entire string. For example, if there is a longer line and it contains the given string, then it should only replace that specific string and not the entire line.
